dist/pmAnimatedBanners.js

```diff
@@ -7222,7 +7222,7 @@ var Layer = function () {
         img.onload = function () {
           return resolve(img);
         };
-        img.src = _this3.data.image;
+        img.src = _this3.data.image + '?' + new Date().getTime();
```

Comment:
changes looks ok
but variables like `_this3` looks bad as for me maybe let's use `self` for example